articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -28,41 +28,48 @@ In this quickstart, you'll learn how to use a code sample that demonstrates how
-> 1. Enter a name for your application and click **Register**.
+> 1. Go to the [Azure portal - Application Registration](https://portal.azure.com/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs).
+> 1. Enter a name for your application.
+> 1. Select **Accounts in any Azure AD directory and personal Microsoft accounts** under Supported Microsoft accounts and click **Register**.
 > 1. Follow the instructions to download and automatically configure your new application for you in one click.
+>
 > [!div renderon="docs"]
-> ### Option 2: Register and manually configure your application and code sample
+> ### Option 2: Register and manually configure your app and then download the code sample
 > #### Step 1: Register your application
-> 
+>
 > 1. To register an application, go to the [Azure portal - Application Registration](https://aka.ms/registeredappsprod) and select **New registration**.
-> 1. Enter a name for your application, add `http://localhost:44316/` in **Reply URL**, and click **Register**.
-> 1. Select **Authentication** menu, set **ID tokens** under **Implicit Grant**, and then select **Save**.
+> 1. Enter a name for your application.
+> 1. Select **Accounts in any Azure AD directory and personal Microsoft accounts** under Supported Microsoft accounts.
+> 1. Under **Redirect URI**, select **Web** and add `http://localhost:30662/`. Click **Register**.
+> 1. Select **Authentication** menu, enable **ID tokens** and **Access tokens** under **Implicit Grant**, and then select **Save**.
 
 > [!div class="sxs-lookup" renderon="portal"]
 > #### Step 1: Configure your application in Azure portal
-> For the code sample for this quickstart to work, you need to add a reply URL as `http://localhost:44316/` and enable **Implicit Grant**.
+> For the code sample for this quickstart to work, you need to add a redirect URI as `http://localhost:30662/` and enable **Implicit Grant**.
 > > [!div renderon="portal" id="makechanges" class="nextstepaction"]
 > > [Make these changes for me]()
 >
 > > [!div id="appconfigured" class="alert alert-info"]
 > > ![Already configured](media/active-directory-javascriptspa/green-check.png) Your application is configured with these attributes
 
-#### Step 2: Download your web server or project
+#### Step 2: Download the project
 
-* [Download the Visual Studio project](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/VisualStudio.zip)
+You can choose one of these options suitable to your development environment.
 * [Download the core project files - for a web server, such as Node.js](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/core.zip)
+* [Download the Visual Studio project](https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/archive/VisualStudio.zip)
+
+Extract the zip file of the project to a local directory such as C://spa.
```

Comment:
For consistence with other quickstarts, I would keep like:

Extract the zip file to a local folder (for example, **C:\Azure-Samples**)